webservice/reference.md

```diff
@@ -225,7 +225,7 @@ When displaying a resource schema in synopsis mode, the API gives some useful in
 | isLinkRewrite | A valid friendly URL. | {{< code >}}/^[_a-zA-Z0-9\-]+$/{{< /code >}} |
 |  | A valid friendly URL (with `PS_ALLOW_ACCENTED_CHARS_URL` enabled). | {{< code >}}/^[_a-zA-Z0-9\x{0600}-\x{06FF}\pL\pS-]+$/u{{< /code >}} |
 | isMpn | A valid mpn (Manufacturer Part Number) 40 characters max. | n/a |
-| isNegativePrice | A valid price value (including negative price) | {{< code >}}/^[-]?[0-9]{1,10}(\.[0-9]{1,9})?$/{{< /code >}} |
-| isPrice | A valid price display method (either PS_TAX_EXC or PS_TAX_INC). | {{< code >}}0|1{{< /code >}} |
+| isNegativePrice | A valid price value (including negative price). | {{< code >}}/^[-]?[0-9]{1,10}(\.[0-9]{1,9})?$/{{< /code >}} |
```

Comment:
Hi @LorisB so you say that `isNegativePrice` is a valid price value? these two have been inverted?

Reply:
`isNegativePrice`'s description and RegEx are ok. I have just added a period at the end of the sentence.
On the other hand, `isPrice`'s definition pertains to `isPriceDisplayMethod`. Maybe that's due to an erroneous copy-paste.